src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_CategoryPickerScreen.tsx

```diff
@@ -64,6 +65,51 @@ export const TicketAssistant_CategoryPickerScreen = ({
     return unsubscribe;
   }, [navigation]);
 
+  function getAdditionalTitleText(userTypeString: string) {
+    switch (userTypeString) {
+      case 'CHILD':
+        return t(_('/ungdom', '/youth', '/ungdom'));
+      default:
+        return '';
+    }
+  }
+  function AdditionalDescription(userTypeString: string) {
```

Comment:
Functions should start with lower case. Also is there a reason why the text is inline in the function instead of in the texts files?